models/Server.js

```diff
@@ -2,460 +2,503 @@ var sql = require("mssql");
 const date = require("date-and-time");
 
 var config = {
-  user: "root",
```

Comment:
Why are you changing the connection string?

Reply:
Sorry forgot to put it back when I was testing I have changed it back